manifests/tasks/regcheck.json

```diff
@@ -0,0 +1,3 @@
+{
+  "description": "Query which .NET versions are installed on Windows Server(s)."
```

Comment:
Is this limited to servers or does it work on desktops too?

Reply:
Should work on all Windows OS's as far as this article is concerned: https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed